### PR TITLE
✅ Testing compatibility with multiple python versions (and actually first CI)

### PR DESCRIPTION
**Pull request by Hi-king**
* introduce CI with github actions
* introduce tox to run test with multiple python versions

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.5, 3.7, 3.8, 3.9]
```
Comment (matsui528) [inline .github/workflows/test.yml:16]: Just want to know why py3.6 is excluded?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
```
Comment (matsui528) [inline .github/workflows/test.yml:23]: The newer version (`setup-python@v2`) is used in the official document. Should we use it?

https://docs.github.com/en/actions/guides/building-and-testing-python#starting-with-the-python-workflow-template

**Review by matsui528 — APPROVED** [review]
Just a few comments. LGTM

File: .github/workflows/test.yml (hunk 1 above)
Comment (Hi-king) [reply .github/workflows/test.yml:16]: @matsui528 I don't intend to enumerate all versions, but just added 3.5 for the lowest version instead

File: .github/workflows/test.yml (hunk 2 above)
Comment (Hi-king) [reply .github/workflows/test.yml]: @matsui528 Cool! I've upgraded both setup-python and checkout actions.

**Pull request merged**